daemon/call.c

```diff
@@ -2595,7 +2597,12 @@ int monologue_offer_answer(struct call_monologue *other_ml, GQueue *streams,
 
 		/* get that many ports for each side, and one packet stream for each port, then
 		 * assign the ports to the streams */
-		em = __get_endpoint_map(media, num_ports, &sp->rtp_endpoint);
+		if (!bridge_count) {
+			em = __get_endpoint_map(media, num_ports, &sp->rtp_endpoint, 0);
+		} else {
+			unsigned int *bridge_port = (unsigned int *)g_queue_pop_head(&call->rtp_bridge_ports);
```

Comment:
Simplified it also taken into consideration that g_queue_pop_head() returns NULL (0) if the queue has no elements.